src/m4st/process_demetr.py

```diff
@@ -2,149 +2,71 @@
 Script for running BLEU, SacreBLEU, and BLASER 2.0 on the DEMETR dataset.
 """
 
-import csv
 import os
+import typing
 
-import numpy as np
 import pandas as pd
 
 from m4st.metrics import (
     BLASERQEScore,
     BLASERRefScore,
+    BleuScore,
     ChrFScore,
     COMETQEScore,
     COMETRefScore,
-    SacreBLEUScore,
 )
 
 
 class ProcessDEMETR:
     def __init__(
         self,
-        output_filepath: os.PathLike | str,
+        output_dir: os.PathLike | str,
         demetr_root: os.PathLike | str,
         metrics_to_use: list,
```

Comment:
```suggestion
        metrics_to_use: list[str],
```

unless the intent is to also be able to pass initialised metrics here, in which case it could be `list[str | Metric]`, perhaps (if also making the change re all classes inheriting from `Metric`)